src/globals/scss/core/_links.scss

```diff
@@ -1,45 +1,66 @@
 @include exports("links") {
-  a:link {
-    color: $govuk-link-colour;
-  }
 
-  a:visited {
-    color: $govuk-link-visited-colour;
-  }
+  .govuk-link {
+    @include govuk-font-regular;
+    @include govuk-focusable-fill;
 
-  a:hover {
-    color: $govuk-link-hover-colour;
-  }
+    // Override the tap highlight colour (the color of the highlight that
+    // appears when a link is tapped on some mobile devices). This is
+    // ever-so-slightly darker than the default.
+    -webkit-tap-highlight-color: rgba($govuk-black, .3);
 
-  a:active {
-    color: $govuk-link-active-colour;
-  }
+    &:link {
+      color: $govuk-link-colour;
+    }
 
-  a {
-    -webkit-tap-highlight-color: rgba($govuk-black, .3);
-  }
+    &:visited {
+      color: $govuk-link-visited-colour;
+    }
 
-  a:focus {
-    outline: $govuk-focus-width solid $govuk-focus-colour;
-    background-color: $govuk-focus-colour;
-  }
+    &:hover {
+      color: $govuk-link-hover-colour;
+    }
+
+    &:active {
+      color: $govuk-link-active-colour;
+    }
+
+    @include mq($media-type: print) {
 
-  @include mq($media-type: print) {
-    a,
-    a:visited {
-      word-wrap: break-word;
+      // When printing, append the the destination URL to the link text, as long
+      // as the URL starts with either `/`, `http://` or `https://`.
+      &[href^="/"],
+      &[href^="http://"],
+      &[href^="https://"] {
+        &::after {
+          content: " (" attr(href) ")";
+          font-size: 90%;
+
+          // Because the URLs may be very long, ensure that they may be broken
+          // at arbitrary points if there are no otherwise acceptable break
+          // points in the line
+          word-wrap: break-word;
+        }
+      }
     }
+  }
 
-    a[href^="/"]:after,
-    a[href^="http://"]:after,
-    a[href^="https://"]:after {
-      content: " (" attr(href) ")";
-      font-size: 90%;
+  // Muted link variant
+  //
+  // Used for secondary links on a page - the link will appear in muted colours
+  // regardless of visited state.
+  .govuk-link--muted {
```

Comment:
Here be dragons:

The intention is that this would be used as a modifier, in addition to the govuk-link class.

```
<a href="#" class="govuk-link govuk-link--muted">Muted link</a>
```

_However_, if a user were to omit the `govuk-link` class, as long as the link was inside another element that set the font they would end up with a link that looked correct _except_ it would not have the focus state.

I don't know what the best approach is here. We _could_ ensure the focus state etc is included in the variants, but we'll end up with a lot of duplication in the CSS and it seems a little weird to have a variant that re-defines everything? Or we could not use modifiers for this sort of thing, and change it to e.g. `.govuk-muted-link` or `.govuk-link-muted`? Thoughts?

Reply:
I think we could move this conversation to the 'should focuses be global' conversation later and not block this PR.

Reply:
Documented as a comment on this card – https://trello.com/c/etYO3raM/598-review-global-focus-stance